tests/testthat/test_check_result_condi.R

```diff
@@ -0,0 +1,121 @@
+context("Check Result Condi")
+
+expect_correct <- function(x) {
+  expect_s3_class(x, "grader_graded")
+  expect_true(x$correct)
+}
+
+expect_wrong <- function(x) {
+  expect_s3_class(x, "grader_graded")
+  expect_false(x$correct)
+}
+
+expect_message <- function(x, message, correct = FALSE) {
+  expect_s3_class(x, "grader_graded")
+  expect_equal(x$correct, correct)
+  expect_true(grepl(message, paste0(x$message, collapse = ""), fixed = TRUE))
+}
+
+test_that("Provide a passing solution. Give the students a fighting chance!", {
+  testthat::expect_error(
+    check_result(
+      grader_args = list(),
+      learnr_args = list(last_value = quote(5), envir_prep = new.env())
```

Comment:
I believe all of the values for `last_value` should be _raw_ values.  Such as `5` (not `quote(5)`)

Reply:
(I know it doesn't matter for these tests in particular, but good to be consistent.)